README.md

```diff
@@ -14,11 +15,24 @@ This is the REST API server for the Mozilla Network Pulse project.
 - [Migrating data](#migrating-data)
 - [Resetting your database because of incompatible model changes](#resetting-your-database-because-of-incompatible-model-changes)
 
+## API Versioning
+
+Any API route that relies on data from the database is versioned (due to its mutability) via its url path so that any changes made to the data responses will not break API clients that rely on specific versions of the data. To get a specific version of the API, add the version after the API prefix (`/api/pulse/`) in the url. For e.g. if you want version 2 (v2) of the API for the `entries` route, you would query the `/api/pulse/v2/entries/` url.
+
+When a new route is added that relies on data in the database, make sure the url pattern setup for that route calls the appropriate versioning function to generate a versioned route. The verioning functions can be found in [utility/urlutils.py](https://github.com/mozilla/network-pulse-api/blob/master/pulseapi/utility/urlutils.py)
+
+#### What happens if you don't specify an API version in the URL?
```

Comment:
With this explanation provided, it seems better to omit the `/api/pulse/v1` prefixes in the rest of the documentation, and instead point to a pre-v2-update tagged version of this README.md once we do release an API v2